### ISSUE TITLE
Formula: Runed Adamantite Rod

### ISSUE THREAD

**Issue by Billshatner** [WoWManiaUK/Redemption#1203]
The Horde vendor for this recipe does not seem to be spawning it.  The NPC in question is called Rungor <Trade Goods>, and is located In Stonebreaker Hold in Terrokar Forest, Outland.  This recipe should spawn roughly once per hour from Rungor.  I have been checking back frequently over the past 2 days time, and have camped at this NPC for well over an hour at one point waiting for the recipe to spawn.  I understand this item is not bind on pickup, and can be bought from the vendor and sold on the auction house.  I also understand players will log out a character next to similar recipe vendors in hopes to buy the item every time it spawns and sell them on auction.  However, the frequency and duration of my visits to Rungor over the past 2 days would not lead me to believe this is the case, and in fact that the respawn timer for this recipe is not functioning properly.  This recipe was also removed sometime after 4.0 and moved to the enchanting trainer, but this should also not be the case as the wowmania server is 3.3.5 and I know there are max level enchanters on the server.

Formula: Runed Adamantite Rod
http://wowwiki.wikia.com/wiki/Formula:_Runed_Adamantite_Rod
http://www.wowhead.com/item=25848/formula-runed-adamantite-rod

Rungor
http://wowwiki.wikia.com/wiki/Rungor
http://www.wowhead.com/npc=18960/rungor

**Comment by fvicaria** [comment]
![image](https://cloud.githubusercontent.com/assets/9414661/26329315/a23ad316-3f3e-11e7-99d1-cd2c3d02530e.png)

**Comment by Billshatner** [comment]
Hello and thank you for the reply.

This picture is frustrating because it clearly shows me standing there,
with the recipe showing spawned for someone else.  I still haven't seen the
recipe on the vendor myself, and wound up buying it from the AH.  I'm not
sure why it does not display on my screen.  I have checked back with Rungor
since receiving this email.  I check his trade window again with all of my
addons enabled, no recipe.  I also turned all addons off and checked him
again, still no recipe.  Even if the recipe is there, I am not able to see
it on my screen.  Here is a screenshot with no addons enabled showing no
recipe.
